.eslintrc

```diff
@@ -9,9 +12,19 @@
     "indent": "off"
   },
   "parserOptions": {
-    "ecmaVersion": 2018
+    "ecmaVersion": 2018,
+    "ecmaFeatures": {
+      "jsx": true
+    }
   },
   "plugins": [
-    "mocha"
+    "mocha",
+    "react"
   ],
+  "settings": {
+    "react": {
+      "pragma": "createElement",
```

Comment:
Just a curiosity about what this pragma means in terms of linting...just ignorance...

Reply:
The pragma tells ESLint what you need to import to use JSX in a file. The default is “React.createElement”, but “createElement” works with both React and Preact. For historical reasons React exports some functions in two different ways.